src/common/ItemForm.js

```diff
@@ -0,0 +1,104 @@
+import { Component } from 'react';
+import './ItemForm.scss';
+
+export default class ItemForm extends Component {
+  state = {
+    name: 'Luckstone',
```

Comment:
you might want to initialize these as empty strings to that they don't auto populate in the form on load. or, alternatively if you want an example you can add them as placeholders instead